Gemfile

```diff
@@ -27,7 +27,7 @@ gem 'redis-rails'
 gem 'redis', '~> 3.0'
 gem 'resque'
 gem 'resque_mailer'
-gem 'resque-retry'
+gem 'resque-scheduler'
```

Comment:
Allows ActiveJob to use `enqueue_at` so we can schedule jobs for a specific time. (See <https://github.com/rails/rails/blob/6a8e7fd8d8cbfa335899153d455082cde906a579/activejob/lib/active_job/queue_adapters/resque_adapter.rb#L34>.)